### PR TITLE
Update tic_tac_toe.py

### PR DESCRIPTION
**Pull request by githubparticipant**
Made updates original file.

### CODE REVIEW

**Comment by harshitashyale** [comment]
Seems like there's no score tracking logic yet?

**Comment by githubparticipant** [comment]
Yeah, I'll add that in the next release

**Comment by harshitashyale** [comment]
Also add multiplayer logic

**Comment by githubparticipant** [comment]
That's out of scope for now.

File: tic_tac_toe.py [hunk 1]
```diff
@@ -2,14 +2,14 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
```
Comment (harshitashyale) [inline tic_tac_toe.py:5]: This should be the other way around - (playerX, playerY)

File: tic_tac_toe.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def checkwinner(playerX, playerY):
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player X has won the game")
+            print("Player Y has won the game")
```
Comment (harshitashyale) [inline tic_tac_toe.py:26]: this should be X

File: tic_tac_toe.py [hunk 3]
```diff
@@ -53,4 +53,3 @@ def checkwinner(playerX, playerY):
             break
 
         turn = 1 - turn
```
Comment (harshitashyale) [inline tic_tac_toe.py:55]:
```suggestion
        turn = turn - 1
```

File: tic_tac_toe.py (hunk 2 above)
Comment (githubparticipant) [reply tic_tac_toe.py:26]: Noted

File: tic_tac_toe.py (hunk 1 above)
Comment (githubparticipant) [reply tic_tac_toe.py:5]: Noted

Comment (githubparticipant) [reply tic_tac_toe.py:5]: "Sound" is not included in this pull request

**Comment by githubparticipant** [comment]
> Made updates original file.

**Comment by githubparticipant** [comment]
Sound feature not included

**Comment by githubparticipant** [comment]
sound [feat] not included in code change.

**Pull request closed without merging**